pkg/oauth_clients/interfaces.go

```diff
@@ -0,0 +1,27 @@
+package oauth_clients
```

Comment:
unless it's a double copy of the previous package or a wip, not sure this should be here?

Reply:
deleted